### DISCUSSION TITLE
Page not updating automatically the events, ajax?

### DISCUSSION THREAD

**Discussion by pauloventurab** [apache/airflow#33072] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### What happened

Hello guys, after resets some pods (scheduler, trigger and worker), once I run some dag, I can't see it refreshing the tasks moving automatically, the logs and etc. I need always to refresh the page to see what is going on and it is not good.

Airflow Version 2.6.1 on GKE Kubernetes

### What you think should happen instead

Restart kubernetes pods

### How to reproduce

I dont know, I want to do know what is the step I need to fix it.

### Operating System

linux

### Deployment

Official Apache Airflow Helm Chart

### Deployment details

helm with some docker modifications just to install some requirements.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by pankajkoti** [comment]
hi @pauloventurab can you check if this is still happening on the latest versions? 2.6.3 is the latest version at the moment and 2.7.0 will be out soon too. For me refresh of logs works fine.

**Comment by potiuk** [comment]
Without reproducible steps, this is at most a discussion and troubleshooting @pauloventurab . You should only open issues if you have good  reason to believe this is a bug in Airlfow (it's explained when you open an issue) - otherwise you should open a discussion. Lack of reproducibility is clear example where it should be a discussion.

You need to deep dive and look at your logs and diagnostics - are your pods running? Are they reporting problems. Is scheduler runnning and showing logs? what kind of deployments you have are your workers running? what your monitoring tells you (You have some monitoring right?) Do you see any errors different than usual? How did you restart your pods? What your kubernetes deployment logs tell you? 

Those are the very basic questions (but this is just a stream of thoughts) that only you can check and provide answers for. 


Just remember, that this is not a helpdesk where people are supposed to solve your problems and get paid for it. This is a forum is where people help in their free time, when they feel like they can help - so if you have problem with your deployment it is on you to provide all the information that can help those people who - in their free time - help people like you to troubleshoot your problems. Spending time on diagnosing and providing useful information on what is going on with your system increases the likelihood that someone will spend few minutes of their time on helping to solve your problem.
